client/src/pages/locations/Form.js

```diff
@@ -367,20 +396,24 @@ const LocationForm = ({ edit, title, initialValues, notesComponent }) => {
         ? response[operation].uuid
         : initialValues.uuid
     })
+    setRelatedObjectUuid(location.uuid)
     // reset the form to latest values
     // to avoid unsaved changes prompt if it somehow becomes dirty
     form.resetForm({ values, isSubmitting: true })
-    if (!edit) {
-      navigate(Location.pathForEdit(location), { replace: true })
-    }
-    navigate(Location.pathFor(location), {
-      state: { success: "Location saved" }
-    })
+    setTimeout(() => {
```

Comment:
`setTimeout()`, really?!

Reply:
Yes, since we can't autoSave while working on the location attachment, I realized that I could bind the attachment to the location relatedly while the page was navigating, and I proceeded that way and thought it was a useful solution since it is not the same in other forms as in the autoSave report.